cmd/eksctl/create.go

```diff
@@ -133,18 +136,22 @@ func doCreateCluster(cfg *eks.ClusterConfig, name string) error {
 	logger.Info("creating EKS cluster %q in %q region", cfg.ClusterName, cfg.Region)
 
 	{ // core action
-		taskErr := make(chan error)
-		// create each of the core cloudformation stacks
-		go ctl.CreateCluster(taskErr)
+		stackManager := ctl.NewStackManager()
+		logger.Info("will create 2 separate CloudFormation stacks for cluster itself and the initial nodegroup")
+		logger.Info("if you encounter any issues, check CloudFormation console first")
```

Comment:
> if you encounter any issues, check CloudFormation console first

Is there anything we could do here to help? I'm thinking the "verification" functionality would help.

Reply:
Yes, for sure. But here I just wanted to make it clear to the user where to go. It's based on feedback from @alexellis, who wasn't aware of CloudFormation, so didn't know that it's the go-to place, he basically ended-up looking all over AWS console and deleting resources one-by-one...

Reply:
We can add an issue to give that extra feedback from cloudformation in the future.

Reply:
#155